packages/x-tree-view/src/internals/useTreeView/useTreeView.ts

```diff
@@ -40,6 +56,10 @@ export const useTreeView = <Plugins extends readonly TreeViewPlugin<TreeViewAnyP
     {} as TreeViewInstance<Signatures>,
   );
   const instance = instanceRef.current as TreeViewInstance<Signatures>;
+
+  const publicAPIRef = useTreeViewApiInitialization<TreeViewPublicAPI<Signatures>>(inParams.apiRef);
```

Comment:
Could `useTreeViewApiInitialization` just return `publicAPI` with the right typing?